lib/trento/activity_logging/activity_log.ex

```diff
@@ -7,6 +7,20 @@ defmodule Trento.ActivityLog.ActivityLog do
 
   @type t() :: %__MODULE__{}
 
+  @derive {
+    Flop.Schema,
+    filterable: [:type, :actor, :inserted_at],
+    sortable: [:type, :actor, :inserted_at],
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** Being the log a stream of entries, I don't see the value in ordering for some criteria different from time.

It's a concern about the product, we might want to raise the issue in the product design phase. 

Anyway, the implementation itself is fine.


‏